src/cpuid.cpp

```diff
@@ -56,7 +56,15 @@ POSSIBILITY OF SUCH DAMAGE.
 
 
 #if TORRENT_HAS_ARM && TORRENT_HAS_AUXV
```

Comment:
in your other patch, you define ``TORRENT_HAS_AUXV`` conditionally based on the target version. With this patch, could that define be removed entirely?

Reply:
I bring it up because it seems these two patches can't be combined cleanly right now, because if you built against an older SDK ``TORRENT_HAS_AUXV`` won't be defined, and you won't try to load the function dynamically.

Reply:
No, I should have specified it. The two last patch I made are aimed at replacing the first patch, so as to be able to compile without disabling the netlink and cpuid features. I'll clean both patch and check again.